transforms/ember-object/__testfixtures__/decorators.output.js

```diff
@@ -38,8 +43,11 @@ class Foo extends EmberObject {
 }
 
 class Comp extends EmberObject {
+  @computed("a", "c")
   @className("enabled", "disabled")
-  isEnabled = false;
+  isEnabled() {
```

Comment:
Yeah, looks little complicated to make a decision whether to make native getter. Should it be just if the property has multiple decorators the value should be either `simple value` or `native getter`? 🤔

Reply:
If it is marked by `@computed`, it will always need to be a native getter. `@className` can also only ever be applied to a native getter, or a class field, neither works with methods.

If it is an `@action` it should be a method. I don't believe there are any other method decorators.

In general, method decorators and accessor/field decorators are pretty mutually exclusive at this point. You don't have decorators that apply to both types, just one or the other.

Reply:
Hmm, that means the this transform is incorrect in the `decorators.output.js`
```
/**
  Computed description
  */
  @computed("fullName", "age", "country")
  description() {
    return `${this.get(
      "fullName"
    )}; Age: ${this.get("age")}; Country: ${this.get("country")}`;
  }
```
It should be 
```
/**
  Computed description
  */
  @computed("fullName", "age", "country")
  get description() {
    return `${this.get(
      "fullName"
    )}; Age: ${this.get("age")}; Country: ${this.get("country")}`;
  }
```

Reply:
I think only `action` and `on` are method decorators

Reply:
ah, and `@observes`, thanks for reminding me about those